apps/testing-javascript/src/lib/secondsToFormattedTime.ts

```diff
@@ -0,0 +1,20 @@
+type Options = {resolveToSeconds: boolean}
+
+export function secondsToFormattedTime(
```

Comment:
Helper function to convert a number of seconds into a human-readable duration format